### DISCUSSION TITLE
Transfer to another computer

### DISCUSSION THREAD

**Discussion by dgcasey0325** [SauceLLC/sauce4zwift#72] [Q&A]
I just installed Sauce 4 Zwift on my Zwift computer and it is working. But I would like to be able to sit at my office computer and do the configuration of the overlay. Doing at my Zwift computer is going to require I stand there for however long it takes me to make whatever changes I desire. Is there a way to customize the overlay on my office system, sitting in my comfy chair and then transfer some setup file to the other computer?

**Comment by mayfield** [comment]
Hello @dgcasey0325,

Yes this is possible.  What you want to do is:

1. Configure the windows as you like on the first computer.
2. Go to the main Settings window and select the **Windows** tab.
3. Find the **Profiles** section and click the little :arrow_down: download button on the row of the Current profile (most likely called "Default").
4. Take the file that is saved and copy it to the other computer.
5. On the other computer navigate to the same settings page and click the **Import from file** button.
6. Select the file exported from computer 1 and you should see a new profile that you can **Activate**
7. *If the screen resolutions are different you'll need to move/size things the first time, but otherwise all the settings should be preserved.
